packages/http-specs/specs/type/model/visibility/main.tsp

```diff
@@ -14,8 +14,10 @@ model VisibilityModel {
   @visibility(Lifecycle.Read)
   readProp: string;
 
+  #suppress "@typespec/http/metadata-ignored" "For test"
```

Comment:
one side question - if the case is not recommanded should we include it in spector?

Reply:
I don't think this really matters here. Also i don't think this warning necessarly is saying its not something you shouldn't do but just a FYI that those properties will not show up in the query but in the body.

In those tests we could change to bodyRoot instead as well.

But I do however thing we should have a test to make sure this case works as expected.